composer/algorithms/fused_layernorm/fused_layernorm.py

```diff
@@ -36,7 +36,11 @@ def from_LayerNorm(layer: torch.nn.Module, module_index: int) -> APEXFusedLayerN
     """Defines a replacement policy from a `torch.nn.LayerNorm` to a `apex.normalization.fused_layer_norm`"""
     assert isinstance(layer,
                       torch.nn.LayerNorm), 'The replacement policy will look for all instances of torch.nn.LayerNorm'
-    return APEXFusedLayerNorm(normalized_shape=layer.normalized_shape, eps=layer.eps)
+    fused_layernorm = APEXFusedLayerNorm(normalized_shape=layer.normalized_shape, eps=layer.eps)
```

Comment:
This still runs at INIT event right?

Reply:
By default, this runs at INIT. So, technically the weight copy (lines 40-42) is technically not needed, because the checkpoint will be loaded afterward and the weights will come from there. However, because other O1 speedups have to be run at Event.POST_LOAD, I added the weight copy so all O1 speedups have the same behavior at POST_LOAD.